decode.arb

```diff
@@ -0,0 +1,41 @@
+morse = { '.-' => 'a',
+          '-...' => 'b',
+          '-.-.' => 'c',
+          '-..' => 'd',
+          '.' => 'e',
+          '..-.' => 'f',
+          '--.' => 'g',
+          '....' => 'h',
+          '..' => 'i',
+          '.---' => 'j',
+          '-.-' => 'k',
+          '.-..' => 'l',
+          '--' => 'm',
+          '-.' => 'n',
+          '---' => 'o',
+          '.--.' => 'p',
+          '--.-' => 'q',
+          '.-.' => 'r',
+          '...' => 's',
+          '-' => 't',
+          '..-' => 'u',
+          '...-' => 'v',
+          '.--' => 'w',
+          '-..-' => 'x',
+          '-.--' => 'y',
+          '--..' => 'z' }
```

Comment:
Please add a method name `decode_char` which translates Morse code symbols into letters. So `decode_char('.-')` should return `A`.